augur/tasks/git/util/facade_worker/facade_worker/facade05repofetch.py

```diff
@@ -114,21 +115,7 @@ def git_repo_initialize(session, repo_git):
             return 
 
         # Create the prerequisite directories
-        return_code = subprocess.Popen([f"mkdir -p {repo_path}"],shell=True).wait()
-#        session.log_activity('Info','Return code value when making directors from facade05, line 120: {:d}'.format(return_code))
-
-
-
-        # Make sure it's ok to proceed
-        if return_code != 0:
-            print("COULD NOT CREATE REPO DIRECTORY")
-
-            update_repo_log(session, row.repo_id,'Failed (mkdir)')
-            session.update_status(f"Failed (mkdir {repo_path})")
-            session.log_activity('Error',f"Could not create repo directory: {repo_path}" )
-
-            raise Exception("Could not create git repo's prerequisite directories. "
-                " Do you have write access?")
+        pathlib.Path(repo_path).mkdir(parents=True, exist_ok=True)
```

Comment:
This is a good change however it should still catch any exceptions Path.mkdir raises and respond like how it did before when we didn't get a return code. It should update its internal logs when it is unable to create the directory.

Reply:
Just put the mkdir line in a try/except block and put the body of the if statement on line 123 in the exception handle. Otherwise looks good.